src/effects/lv2/LV2FeaturesList.cpp

```diff
@@ -178,9 +186,11 @@ bool LV2FeaturesList::CheckFeatures(const LilvNode *subject, bool required)
             /* Supported but handled in LV2Wrapper */
          }
          else if (required) {
-            const auto end = mFeatures.end();
-            supported = (end != std::find_if(mFeatures.begin(), end,
-               [&](auto &feature){ return strcmp(feature.URI, uri) == 0; }));
+            auto features = GetFeaturePointers();
```

Comment:
GetFeaturePointers() builds a vector of pointers, as you may recall from reviewing an earlier PR.

I could write
`const auto &features = ...`

and it would also compile, and the reviewer might not then have to pause and think about it.

Reply:
As for threads, this should be done only in construction and initializations of the plug-in, or its instances, or its validators, in the main thread